ide/hudson/src/org/netbeans/modules/hudson/Bundle.properties

```diff
@@ -17,7 +17,7 @@
 
 OpenIDE-Module-Display-Category=Base IDE
 OpenIDE-Module-Long-Description=\
-    Helps browse and manage installations of the Hudson continuous integration server (http://hudson-ci.org/). \
+    Helps browse and manage installations of the Jenkins continuous integration server (http://hudson-ci.org/). \
     http://wiki.netbeans.org/HudsonInNetBeans has more information.
```

Comment:
Should we remove this line because http://wiki.netbeans.org/HudsonInNetBeans is no longer available?

Reply:
for this we could later redirect or rewrite content.

To have an overview
https://web.archive.org/web/20190816094055/http://wiki.netbeans.org/HudsonInNetBeans